com.microsoft.mrtk.graphicstools.unity/Samples.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
This meta file can be deleted since we check-in the Samples directory as "Samples~"